### PR TITLE
[ticket/16538] Add MySQL 8 tests

### PR DESCRIPTION
**Pull request by rxu**
Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

<a href="https://tracker.phpbb.com/browse/PHPBB3-16538">PHPBB3-16538</a>.
<a href="https://tracker.phpbb.com/browse/PHPBB3-16535">PHPBB3-16535</a>.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,6 +35,9 @@ services:
   - mysql
   - memcached
 
+before_install:
+  - travis/setup-mysql8.sh $MYSQL8
```
Comment (marc1706) [inline .travis.yml:39]: Script execution should be added to setup-phpbb.sh:
https://github.com/phpbb/phpbb/blob/master/travis/setup-phpbb.sh

Comment (rxu) [reply .travis.yml:39]: MySQL got upgraded on `before_install` part so `setup-phpbb.sh` is already running on MySQL 8, or isn't it?

Comment (rxu) [reply .travis.yml:39]: Oh, I guess I see, you mean move it from there to `setup-phpbb.sh`. ok.

**Comment by 3D-I** [comment]
Interesting work.

**Comment by rxu** [comment]
Well, the core itself works fine with it so far. But bugs are possible just like for everything else :)

**Comment by 3D-I** [comment]
I luuv tests :rofl:

**Review by marc1706 — APPROVED** [review]

**Pull request merged**